Sources/VLCSettingsController.m

```diff
@@ -171,4 +226,45 @@ - (void)updateForPasscode:(NSString *)passcode
         [self.navigationController.presentedViewController dismissViewControllerAnimated:YES completion:nil];
     }
 }
+
+# pragma mark - IASKAppSettingsViewController override
+
+- (UITableViewCell*)newCellForIdentifier:(NSString*)identifier {
+    UITableViewCell *cell = nil;
+    if ([identifier isEqualToString:kIASKPSToggleSwitchSpecifier]) {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:kIASKPSToggleSwitchSpecifier];
+        cell.accessoryView = [[IASKSwitch alloc] initWithFrame:CGRectMake(0, 0, 79, 27)];
```

Comment:
Can we write cell subclasses for this and not cram everything in this method and viewcontroller? Also magic numbers. By having subclass in an extra file we should also be able to remove some of the above imports I reckon, which will make the entire thing more readable